powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/converter/RegistrationConverter.java

```diff
@@ -64,7 +64,8 @@ public AuthenticatorDetail convert(RegistrationChallenge challenge, Registration
             authenticatorDetail.setMaxFailedAttempts(5L);
             return authenticatorDetail;
         } catch (JsonProcessingException e) {
-            throw new RuntimeException(e);
+            logger.warn(e.getMessage(), e);
```

Comment:
General `RuntimeException` is not good, but either returning `null`, I would suggest to change it into `Optional`.

Reply:
Sure, we can use `Optional`.